source/nodejs/adaptivecards-controls/src/inputwithpopup.ts

```diff
@@ -177,11 +177,14 @@ export abstract class InputWithPopup<TPopupControl extends PopupControl, TValue>
     private _popupControl: TPopupControl;
     private _placeholderText: string;
     private _value: TValue;
```

Comment:
Can we remove Theme and Size from other containers? 
Size only means something to Berlin.
Theme is a little tricky, but we should not add it until you merge dark/light their contain into one.

![image](https://user-images.githubusercontent.com/6290692/186984899-0f35de02-2fa1-46f7-9eed-55ef7b8692e4.png)

Reply:
@jwoo-msft @paulcam206 What do you think?

Reply:
I like how Teams and Outlook have their "Emulate Device" only show up on their containers. I agree that other host apps like Viva and Bot Framework wont need a size dropdown and this should be mainly for the berlin containers.

Reply:
To clarify, the size dropdown has already been updated and is only displayed for the Berlin container. The theme container is visible on all of the containers with the idea that Microsoft Teams, Cortana Skills, and Viva Connections will all use it (to be added in the my next PR).

Reply:
Resolving and created an issue for using the drop-down menu for all containers with a light and dark theme: https://github.com/microsoft/AdaptiveCards/issues/7813